addons/website_sale_delivery/controllers/main.py

```diff
@@ -85,44 +85,45 @@ def process_express_checkout(self, shipping_address, shipping_option, **post):
         :param dict post: other information used in the parent route.
         :return int: The order's partner id.
         """
-        order_sudo = request.website.sale_get_order()
```

Comment:
Maybe return when `not shipping_adress or not shipping option`, so you don't over indent everything... :thinking: 
Here you have a if in a if in a if, seems a bit too much. And this'll decreased the diff too =]